src/importexport/guitarpro/internal/gtp/gp67dombuilder.cpp

```diff
@@ -22,20 +22,20 @@ void GP67DomBuilder::buildGPDomModel(QDomElement* qdomElem)
     // Currently ignored
     QDomNode gpversion, encoding;
 
-    std::map<QString, QDomNode*> nodeMap =
+    std::map<mu::String, QDomNode*> nodeMap =
     {
-        { "GPRevision",    &revision },
-        { "Score",               &scoreNode },
-        { "MasterTrack",   &masterTrack },
-        { "Tracks",        &eachTrack },
-        { "MasterBars",    &masterBars },
-        { "Bars",                &bars },
-        { "Voices",        &voices },
-        { "Beats",               &beats },
-        { "Notes",               &notes },
-        { "Rhythms",       &rhythms },
-        { "GPVersion",   &gpversion },
-        { "Encoding",    &encoding }
+        { u"GPRevision",    &revision },
+        { u"Score",               &scoreNode },
```

Comment:
I believe that it is possible to correct the alignment

Reply:
Yep. I made it